src/main/resources/db/changelog/changes/05-create-book-category-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-category-table
+      author: denis
+      changes:
+        - createTable:
+            tableName: book_category
+            columns:
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: category_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
add newLine and what if you insert book with non-existing categories? you need to add foreignKey and primaryKey

Reply:
i fix it in next repo and other liquibase files

Reply:
Why not fix it in this PR? You are trying to merge broken code